extensions/GitExtension/FileExplorerGitIntegration/Models/RepositoryWrapper.cs

```diff
@@ -171,6 +172,26 @@ public string GetFileStatus(string relativePath)
         return statusString;
     }
 
+    private string GetOriginalPath(string relativePath)
+    {
+        _statusCache.Status.FileEntries.TryGetValue(relativePath, out var status);
+        if (status is null)
+        {
+            return relativePath;
+        }
+
+        if (status.Status.HasFlag(FileStatus.RenamedInIndex))
+        {
+            return status.RenameOldPath ?? relativePath;
+        }
+        else if (status.Status.HasFlag(FileStatus.RenamedInWorkdir))
+        {
+            return status.RenameOldPath ?? relativePath;
+        }

```

Comment:
both conditions seems to be returning same thing. If that's intended, maybe combine in single if?